src/main/java/net/minecraftforge/registries/DeferredRegister.java

```diff
@@ -210,15 +165,10 @@ public Supplier<IForgeRegistry<T>> makeRegistry(final Supplier<RegistryBuilder<T
     }
 
     /**
-     * Creates a tag key based on the current modid and provided path as the location and the registry name linked to this DeferredRegister.
-     * To control the namespace, use {@link #createTagKey(ResourceLocation)}.
+     * Calls {@link #createTagKey(ResourceLocation)} with the given path, using the current modid as the namespace.
```

Comment:
I have similar thoughts here as I do to the variants of `#create`. My thoughts are that we shouldn't require people to go through a javadoc chain to figure out what the current method does. Therefore, duplication makes more sense.

Reply:
In general I prefer `@see` because it centralizes the information - this is valuable in some cases, but not this one (as the information proxied is very limited, about 5 words or so).